### PR TITLE
OCPBUGS-17073: Revert "Merge pull request #7205 from rna-afk/azure_managed_by_tag"

### PR DESCRIPTION
**Pull request by rna-afk**
This reverts commit 87fbf266448b0d1749e01ffb8c3df9f7a71521e4, reversing changes made to 26a0d5cfeb12439901174a5c64adc33e3364de92.

### CODE REVIEW

**Comment by r4f4** [comment]
/lgtm cancel
We need tf-azure provider >= 3.43.0 for the confidential VM feature

File: terraform/providers/azurerm/go.mod [hunk 1]
```diff
@@ -2,14 +2,14 @@ module github.com/openshift/installer/terraform/providers/azurerm
 
 go 1.18
 
-require github.com/hashicorp/terraform-provider-azurerm v1.44.1-0.20220819162217-6be27548882d // v3.19.1
+require github.com/hashicorp/terraform-provider-azurerm v1.44.1-0.20230210023348-72f0799cec36 // v3.19.1
```
Comment (r4f4) [inline terraform/providers/azurerm/go.mod:5]: Don't forget to update the comment with the current version.

**Review by r4f4 — COMMENTED** [review]
The terraform chagnes need to be vendored in.

**Comment by MayXuQQ** [comment]
MAG cluster install succeed with [pre-merge build](https://storage.googleapis.com/origin-ci-test/logs/release-openshift-origin-installer-launch-aws-modern/1691628803916304384/build-log.txt)

**Review by MayXuQQ — APPROVED** [review]

**Comment by r4f4** [comment]
@rna-afk do you plan on squashing https://github.com/openshift/installer/pull/7412/commits/d9574929948763c86ea501516b151a1064ceb419 into https://github.com/openshift/installer/pull/7412/commits/a34b5355152bc36bc58d205439e29e98b8a7e5d9?

**Comment by rna-afk** [comment]
@r4f4 squashed

**Pull request merged**